src/test/java/bogdanSandu/tests/tema8/pageObjects/DynamicButtonsDisabledPage.java

```diff
@@ -0,0 +1,91 @@
+package bogdanSandu.tests.tema8.pageObjects;
+
+import bogdanSandu.tests.tema8.tests.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class DynamicButtonsDisabledPage extends BaseTest {
```

Comment:
Please remove the extends BaseTest part. BaseTest it's the base only for test classes, not page object classes. You have a driver defined here in page class. Thanks!